modules/stats/stats.py

```diff
@@ -66,6 +69,19 @@ def calculate_trade_durations(closed_trades):
     return avg_trade_duration, longest_trade_duration, shortest_trade_duration
 
 
+def get_market_drawdown(pairs: list, data_dict: dict) -> dict:
+    market_drawdown = {}
+    total_drawdown = [0] * len(data_dict[pairs[0]])
+    for pair in pairs:
+        values_list = [data_dict[pair][d].get('close') for d in data_dict[pair]]
+        coin_values = DataFrame(values_list, columns=['value'])
+        market_drawdown[pair] = get_max_drawdown_ratio(coin_values)
+        values_list = [x / values_list[0] for x in values_list]
+        total_drawdown = map(lambda x, y: x + y, total_drawdown, values_list)
+    market_drawdown['all'] = get_max_drawdown_ratio(DataFrame(total_drawdown, columns=['value']))
+    return market_drawdown
```

Comment:
This looks like it works! Performance wise this might get a bit slow when testing longer periods though (we'll cross that bridge when we get there, using `df` might be a bit quicker ). 

What do you think of the following names?,  
```suggestion
    market_drawdown = {}
    pair_profit_sum = [0] * len(data_dict[pairs[0]])
    for pair in pairs:
        close_prices = [data_dict[pair][d].get('close') for d in data_dict[pair]]
        close_prices_df = DataFrame(values_list, columns=['value'])
        market_drawdown[pair] = get_max_drawdown_ratio(close_prices_df)
        profit_ratios = [x / values_list[0] for x in values_list]
        pairs_profit_ratios_sum = map(lambda x, y: x + y, pair_profit_sum, profit_ratios)
    market_drawdown['all'] = get_max_drawdown_ratio(DataFrame(total_drawdown, columns=['value']))
    return market_drawdown
```

Reply:
Variables are renamed! Thanks for the suggestions!